include/TrustWalletCore/TWBlockchain.h

```diff
@@ -65,6 +65,8 @@ enum TWBlockchain {
     TWBlockchainSui = 50,
     TWBlockchainGreenfield = 51,
     TWBlockchainInternetComputer = 52,
+    TWBlockchainNativeEvmos = 55, // Cosmos
+    TWBlockchainNativeInjective = 56, // Cosmos
```

Comment:
how it was working before?

Reply:
Before, we just checked the coin type, and used different public key prefix types for NativeEvmos and NativeInjective coins.
It smells like a bad pattern, so in Rust provide `CosmosContext` where the `CosmosContext::PublicKey` solves this problem.